tests/infinity_constants.h

```diff
@@ -0,0 +1,14 @@
+#ifndef TESTS_INFINITY_CONSTANTS_H
+#define TESTS_INFINITY_CONSTANTS_H
+
+#include <limits>
+
+namespace flatbuffers {
+namespace tests {
+// Shortcuts for the infinity.
+inline float infinity_f() { return std::numeric_limits<float>::infinity(); }
```

Comment:
This function are so over engineered :)

Reply:
If we were at c++17, this could be a simple `inline constexpr`.

Reply:
static const auto or static constexpr is much simpler than this functions

Reply:
If you make it 'static' constexpr in a header, you duplicate the symbol in each compilation unit including it.

Reply:
how about just a macro instead?